managing-docs.md

```diff
@@ -7,6 +7,7 @@ We use the following services for storing information:
 - **[Intra](https://intra.niteoweb.com)** - Project documentation, team members profiles, reports and anything else that cannot be publicly published on the Handbook.
 - **[collective.permalink](https://pypi.python.org/pypi/collective.permalink)** - an extension for **[Intra](https://intra.niteoweb.com)** that creates a persistent link to the Intra's content that should never change even if you rename or move it. Use it every time you need to reference some Intra content from the outside of Intra, like from the Handbook. To get a permalink click on Plone toolbar `Actions` > `Permalink (to Clipboard)`.
 - **[Docs](http://docs.niteoweb.com)** - Technical documentation for projects, generated automatically by Sphinx on every code change. Also a ton of books and videos from our past events.
+- **[Grafana](niteo.grafana.net)** - Visualizations and statistics of the company and projects.
```

Comment:
I would just say "Company and project dashboards." because dashboard is a pretty standard term.

Reply:
Dashboard was the word that was eluding me :stuck_out_tongue:

Reply:
Fixed